includes/class-wc-calypso-bridge-setup.php

```diff
@@ -4,7 +4,7 @@
  *
  * @package WC_Calypso_Bridge/Classes
  * @since   1.0.0
- * @version 2.2.13
+ * @version x.x.x
```

Comment:
Is this correct or does it need to be an actual version number? I'm not super familiar with how the versioning works.

Reply:
Yes, we bump the `x.x.x`s when we're about to release the version, as we don't know in advance in which version this PR will make it.